zoo-css-challenge/index.html

```diff
@@ -64,8 +65,35 @@ <h3>Top 5 Tiger Facts</h3>
 
 		<section class="giraffe">
 			<h2>Giraffe</h2>
+			<ul>
+				<li>
+				  Tigers are the largest cat species in the world.
+				</li>
+				<li>
+				  Tigers are easily recognizable with their dark vertical stripes and
+				  reddish/orange fur.
+				</li>
+				<li>The Bengal tiger is the most common tiger.</li>
+				<li>Tigers live between 20-26 years in the wild.</li>
+				<li>
+				  Unlike most other cats, tigers are great swimmers and actually like
+				  the water.
+				</li>
+			  </ul>
+			</section>
 
 			<p>Giraffes are the world's tallest mammals, thanks to their towering legs and long necks. A giraffe's legs alone are taller than many humans—about 6 feet . These long legs allow giraffes to run as fast as 35 miles an hour over short distances and cruise comfortably at 10 miles an hour over longer distances.</p>
+			<section class="giraffe">
+				<h2>Giraffe</h2>
+		
+					<h3>Behavior</h3>
```

Comment:
```suggestion
```
We already have `<section class="giraffe"> <h2>Giraffe</h2>` on lines 66-67 above so we don't need to duplicate them here. We also already have `<h3>Behavior</h3>` below on line 98 so we don't need it here.